packages/react-composites/src/composites/CallComposite/ConfigurationScreen.tsx

```diff
@@ -50,6 +53,16 @@ export const ConfigurationScreen = (props: ConfigurationScreenProps): JSX.Elemen
     </Stack.Item>
   );
 
+  const callDescription = (
+    /*locale.strings.call.configurationPageCallDetails &&*/ <Stack.Item
+      className={mobileView ? callDetailsStyleMobile : callDetailsStyleDesktop}
+    >
+      {
+        'Some details about the call that span more than line - many many lines because the paper bag princess would not stand for such few lines! Could you even imagine 😲'
```

Comment:
This is temporary, right? :D

Reply:
Oops yes that was just for testing hahaha, thanks for catching that